lib/modules/hover.js

```diff
@@ -413,22 +413,16 @@ addModule('hover', function(module, moduleID) {
 	HoverDropdownList.prototype = new Hover();
 	HoverDropdownList.prototype.templateName = 'RESHoverDropdownList';
 	HoverDropdownList.prototype._updatePosition = function($container) {
-		var XY = RESUtils.getXYpos(this._target);
-
-		var height = $(this._target).height();
-
-		var containerWidth = $container.outerWidth();
-
-		var left = XY.x;
-		if (left + containerWidth > window.innerWidth) {
-			left = window.innerWidth - containerWidth - 10;
-		}
-
-		$container.css({
-			top: XY.y + (height * 1.1) + 10,
-			left: left,
-			position: 'fixed',
-			zIndex: 2147483646
-		});
+		var trigger = $(this._target)[0];
+		var menu = $container[0];
```

Comment:
Do we need to avoid jQuery here? The following appears to work the same:

```javascript
var pos = RESUtils.getXYpos(this._target);
var t = pos.y + this._target.offsetHeight,
    r = document.body.scrollWidth - pos.x - this._target.offsetWidth,
    l = pos.x;
$container.css({
	right: l + $container.outerWidth() < document.body.scrollWidth ? 'auto' : r,
	left: l + $container.outerWidth() < document.body.scrollWidth ? pos.x : 'auto',
	zIndex: 2147483646,
	position: 'fixed',
	top: t + 2
});
```